editor/src/dev/dev.clj

```diff
@@ -903,6 +907,10 @@
              (namespaced-class-symbol CursorRange)
              (partial object-data-pprint-handler nil code.data/cursor-range-print-data)
 
+             (namespaced-class-symbol RenderPass)
+             (fn render-pass-pprint-handler [printer ^RenderPass render-pass]
+               (fmt-doc printer (symbol "pass" (.nm render-pass))))
+
```

Comment:
Pretty-print `RenderPass` values as names like `pass/opaque-selection` instead of a map of all its internals. We have a limited global set of passes, and this makes it a lot easier to inspect scene data, where these can be used as map keys.